packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -81,7 +83,18 @@ export function PanelChrome({
 
   // To Do rely on hoverHeader prop for header, not separate props
   // once hoverHeader is implemented
-  const hasHeader = title.length > 0 || leftItems.length > 0;
+  //
+  // Backwards compatibility for having a designated space for the header
+
+  const hasHeader =
```

Comment:
I thought we decided this logic was going to be outside PanelChrome and we instead pass a prop hoverHeader, that way we can have a panel with title and description/links with no header (shown on hover above it )

But if that change is for a future PR where the hover header is added , if so ignore comment :)

But I see we have the prop named hoverHeader so a bit confused

Reply:
Yes, `hoverHeader` feature will be tackled on a different PR. We added this logic here to:

1. Keep backward compatibility with the current `PanelChrome`:
	- When `title` and `leftItems` are both undefined, we do NOT have a header.
	- `When `title` is undefined, but we have `leftItems`, then we have a header. 
	
	In this scenario, we can't rely on `hoverHeader` because by default this prop is `false`

2. Put always a header until we implement the `hoverHeader`, because we are targeting this PR to 9.4 and we are not sure `hoverHeader` will be ready for that time,.